addons/event_sale/models/event.py

```diff
@@ -133,6 +133,7 @@ def _compute_seats(self):
                         WHERE event_ticket_id IN %s AND state IN ('draft', 'open', 'done')
                         GROUP BY event_ticket_id, state
                     """
+            self.env['event.registration'].flush(['event_id', 'state'])
```

Comment:
Why don't you flush `event_ticket_id`, too?
```suggestion
            self.env['event.registration'].flush(['event_id', 'event_ticket_id', 'state'])
```

Reply:
Updated